tests/test_noise.py

```diff
@@ -187,19 +178,6 @@ def test_random_noise(self):
             noisy_pulses[2].coherent_noise[0].tlist,
             np.linspace(1, 6, int(5/0.2) + 1))
 
-    def test_user_defined_noise(self):
```

Comment:
We should add a test that falling back to `.get_noisy_dynamics()` works as expected.

Reply:
Ah, we do already! Woot.